test/hw9/test_hw9_1.py

```diff
@@ -0,0 +1,60 @@
+"""TEST Homework 9.1."""
+from pathlib import Path
+from typing import List, Union
+
+import pytest
+
+from hw.hw9.hw_9_1 import merge_sorted_files, merge_sorted_inf_seq
+
+
+@pytest.fixture()
+def double_path() -> List[Union[Path, str]]:
+    path1 = Path("test/hw9/test_task1/double1.txt")
```

Comment:
We can split repeated lines here and below using tuple with raw paths and generate iterable with Path objects